apps/full-site-editing/full-site-editing-plugin/premium-content/premium-content-dom.php

```diff
@@ -59,7 +59,7 @@ public function remove_element_by_class( string $class ) {
 	 * @return string
 	 */
 	public function to_html() {
-		return $this->doc->saveHTML();
+		return utf8_decode( $this->doc->saveHTML() );
```

Comment:
🤣  nice change @mmtr !

Reply:
Which was actually doing nothing, so reverted in 858e323 😅 I had some other changes locally that I didn't commit in 7a53d85 because I thought they were not needed, but it turned out they were the ones handling the encoding.